Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.5.3-slim-stretch
+
+WORKDIR /app
+COPY Gemfile* ./
+
+RUN apk update && apk add --no-cache \
```

Comment:
we might not need all these packages yet, you might remove this command.

abtw, `apk` is for alpine packages, for debian based images we use `apt`, like `apt-get install package-name`